src/Core/src/Platform/iOS/TabbedViewExtensions.cs

```diff
@@ -9,6 +9,20 @@ namespace Microsoft.Maui.Platform
 {
 	internal static class TabbedViewExtensions
 	{
+		internal static void DisableiOS18ToolbarTabs(
+			this UITabBarController tabBarController)
+		{
+			// Should apply to iOS and Catalyst
+            if (OperatingSystem.IsMacCatalystVersionAtLeast(18,0,-1) || //https://github.com/xamarin/xamarin-macios/issues/21390
+				OperatingSystem.IsIOSVersionAtLeast(18,0))
+            {
+				tabBarController.TraitOverrides.HorizontalSizeClass = UIUserInterfaceSizeClass.Compact;
+                tabBarController.Mode = UITabBarControllerMode.TabSidebar;
+                tabBarController.Sidebar.Hidden = true;
```

Comment:
```
D:\a\_work\1\s\src\Core\src\Platform\iOS\TabbedViewExtensions.cs(20,34): error CS1061: 'UITabBarController' does not contain a definition for 'Mode' and no accessible extension method 'Mode' accepting a first argument of type 'UITabBarController' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net8.0-ios]
D:\a\_work\1\s\src\Core\src\Platform\iOS\TabbedViewExtensions.cs(20,41): error CS0103: The name 'UITabBarControllerMode' does not exist in the current context [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net8.0-ios]
D:\a\_work\1\s\src\Core\src\Platform\iOS\TabbedViewExtensions.cs(21,34): error CS1061: 'UITabBarController' does not contain a definition for 'Sidebar' and no accessible extension method 'Sidebar' accepting a first argument of type 'UITabBarController' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net8.0-ios]
D:\a\_work\1\s\src\Core\src\Platform\iOS\TabbedViewExtensions.cs(22,34): error CS1061: 'UITabBarController' does not contain a definition for 'TabBarHidden' and no accessible extension method 'TabBarHidden' accepting a first argument of type 'UITabBarController' could be found (are you missing a using directive or an assembly reference?) [D:\a\_work\1\s\src\Core\src\Core.csproj::TargetFramework=net8.0-ios]
    36 Warning(s)
    16 Error(s)
```

Reply:
That's expected, https://github.com/dotnet/maui/pull/25140 needs to be merged first to get the newest SDK with the bindings.